ws_esp32_s3_matrix.json

```diff
@@ -0,0 +1,48 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s3_out.ld",
+      "partitions": "default.csv",
+      "memory_type": "qio_qspi"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DARDUINO_WS_ESP32_S3_MATRIX",
+      "-DBOARD_HAS_PSRAM",
+      "-DBOARD_FLASH_MODE=QIO",
```

Comment:
remove flash mode and type here

Reply:
OK, will delete the entire `extra_flags:` section.